packages/rest/src/validate.ts

```diff
@@ -0,0 +1,9 @@
+import { Program } from "@cadl-lang/compiler";
+import { getAllRoutes } from "./route.js";
+
+export function $onValidate(program: Program) {
```

Comment:
hhm, this is now making the samples fail because we have 

```
@added("v2")
  @post
  walkDog(...ApiVersionParam): OkResponse<{}>;

  @removed("v2")
  @post
  walkCat(...ApiVersionParam): OkResponse<{}>;
```

with both of those being at the same endpoint